apps/web/app/(app)/environments/[environmentId]/surveys/PreviewSurvey.tsx

```diff
@@ -17,7 +18,7 @@ import { useEffect, useRef, useState } from "react";
 interface PreviewSurveyProps {
   setActiveQuestionId: (id: string | null) => void;
   activeQuestionId?: string | null;
-  questions: Question[];
+  questions: Question[] | TSurveyQuestion[];
```

Comment:
can we now fully commit to the new types or is anything still missing to do this? 😊

Reply:
Its is now not required as a prop after new changes :)